unittests/test-contracts/test_api/test_api.hpp

```diff
@@ -29,7 +29,7 @@ namespace eosio { class transaction; }
 
 extern "C" {
     __attribute__((eosio_wasm_import))
-    void set_action_return_value(const char*, size_t);
+    void set_action_return_value(void*, size_t);
```

Comment:
Is this change to fix compilation error, or to be consistent with `set_action_return_value` declaration somewhere else?

Reply:
The function is declared in CDT `libraries/eosiolib/capi/eosio/action.h`. You would get declaration conflict if they don't match.